exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/first_steps.ipynb

```diff
@@ -0,0 +1,820 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "91051dad-8d45-435f-b2c9-7572e4050f58",
+   "metadata": {},
+   "source": [
+    "# First Steps\n",
+    "\n",
+    "This notebook guides you through the first steps using Jupyter notebooks with Exasol.\n",
+    "\n",
+    "The notebook demonstrates connecting to an Exasol database instance and using some of its features. \n",
+    "\n",
+    "## 1. Open Secure Configuration Storage\n",
+    "\n",
+    "First we need to open the Secure Configuration Storage (SCS) containing the connection information such as the database host, user, password, etc."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5970a003-896d-4f06-83ee-be98129c29d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run utils/access_store_ui.ipynb\n",
+    "display(get_access_store_ui())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d15075fa-8c9d-4e60-82d9-28d92d38ee0e",
+   "metadata": {},
+   "source": [
+    "## 2. Using JupySQL\n",
+    "\n",
+    "First we will activate the [JupySQL](https://jupysql.ploomber.io) magics:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "73122275-3f4a-4981-b8b3-ee6c49734996",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%run utils/jupysql_init.ipynb"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "342a7800-253d-43a0-9bd7-89932876317d",
+   "metadata": {},
+   "source": [
+    "In the background JupySQL uses SQLAlchemy, see also the [demo section on SQLAlchemy](#4.-SQLAlchemy) below.\n",
+    "\n",
+    "### 2.1 Create Database Tables\n",
+    "\n",
+    "We will use JupySQL to create 2 database tables but other sections will use the tables, too:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a7f5ec4-10f3-49e8-a6e4-48552a760542",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%sql\n",
+    "CREATE\n",
```

Comment:
why do we have here a line break

Reply:
This was probably a left-over from restoring my formatting after accidentally hitting "format SQL" in the UI - See next push removing the unwanted additional line break.